packages/server/package.json

```diff
@@ -36,29 +36,22 @@
     "@types/jest": "^27.0.1",
     "@types/json2csv": "^5.0.3",
     "@types/jsonwebtoken": "^8.3.0",
-    "@types/linkify-it": "^3.0.2",
     "@types/mime-types": "^2.1.0",
     "@types/ms": "^0.7.30",
     "@types/node": "16",
     "@types/nodemailer": "^6.4.4",
-    "@types/react-relay": "^11.0.2",
     "@types/relay-runtime": "^11.0.4",
     "@types/sharp": "^0.29.4",
     "@types/string-similarity": "^3.0.0",
     "@types/stripe": "^6.32.0",
     "@types/uuid": "^8.3.3",
+    "@pgtyped/cli": "^0.11.0",
```

Comment:
I moved this to `devDependencies`. It didn't seem like something that needed to be in the prod deps.